Structure_Engine/Query/Description.cs

```diff
@@ -0,0 +1,232 @@
+using BH.oM.Base;
+using BH.oM.Structure.SurfaceProperties;
+
+namespace BH.Engine.Structure
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods - Profiles                 ****/
+        /***************************************************/
+
+        public static string Description(this ISectionProfile profile)
+        {
+            return "I " + profile.Height + "x" + profile.Width + "x" + profile.WebThickness + "x" + profile.FlangeThickness;
+        }
+
+        /***************************************************/
+
+        public static string Description(this BoxProfile profile)
+        {
+            return "Box " + profile.Height + "x" + profile.Width + "x" + profile.Thickness;
+        }
+
+        /***************************************************/
+
+        public static string Description(this AngleProfile profile)
+        {
+            return "Angle " + profile.Height + "x" + profile.Width + "x" + profile.WebThickness + "x" + profile.FlangeThickness;
+        }
+
+        /***************************************************/
+
+        public static string Description(this ChannelProfile profile)
+        {
+            return "Channel " + profile.Height + "x" + profile.FlangeWidth + "x" + profile.WebThickness + "x" + profile.FlangeThickness;
+        }
+
+        /***************************************************/
+
+        public static string Description(this CircleProfile profile)
+        {
+            return "Circle " + profile.Diameter;
+        }
+
+        /***************************************************/
+
+        public static string Description(this FabricatedBoxProfile profile)
+        {
+            return "FabBox " + profile.Height + "x" + profile.Width + "x" + profile.WebThickness + "x" + profile.TopFlangeThickness + "x" + profile.BotFlangeThickness;
+        }
+
+        /***************************************************/
+
+        public static string Description(this GeneralisedFabricatedBoxProfile profile)
+        {
+            return "GenFabBox " + profile.Height + "x" + profile.Width + "x" + profile.WebThickness + "x" + profile.TopFlangeThickness + "x" + profile.BotFlangeThickness + "x" + profile.TopLeftCorbelWidth + "x" + profile.BotLeftCorbelWidth;
+        }
+
+        /***************************************************/
+
+        public static string Description(this KiteProfile profile)
+        {
+            return "Kite " + Math.Round(profile.Angle1,2) + "x" + profile.Width1 + "x" + profile.Thickness;
+        }
+
+        /***************************************************/
+
+        public static string Description(this FabricatedISectionProfile profile)
+        {
+            return "FabI " + profile.Height + "x" + profile.WebThickness + "x" + profile.TopFlangeWidth + "x" + profile.BotFlangeWidth + "x" + profile.TopFlangeThickness + "x" + profile.BotFlangeThickness;
```

Comment:
Would change this to:

`FabI " + profile.Height + "x" + profile.TopFlangeWidth + "x" + profile.BotFlangeWidth + "x" + profile.WebThickness + "x" + profile.TopFlangeThickness + "x" + profile.BotFlangeThickness;`

I.e. main parameters first (heights, and widths etc) then corresponding thicknesses.